templates/zerver/integrations.html

```diff
@@ -2940,6 +2940,93 @@
       their own Zulip account email in the webhook URL.</p>
     </div>
 
+    <div id="wordpress" class="integration-instructions">
+
+      <p>See WordPress blog notifications in Zulip!</p>
```

Comment:
After moving to separeate file it'll be important to have formatting like: (in case where between markups is more than one line of text)
```
<p>
    one line
    second line
</p>
```